packages/core/upload/admin/src/components/TableList/tests/TableRows.test.js

```diff
@@ -154,6 +161,14 @@ describe('TableList | TableRows', () => {
       expect(getByRole('checkbox', { name: 'Select folder 1 folder', hidden: true })).toBeChecked();
     });
 
+    it('should disabled select folder checkbox when users do not have the permission to update', () => {
```

Comment:
```suggestion
    it('should disable select folder checkbox when users do not have the permission to update', () => {
```
sorry a bit nit picky

Reply:
learning english through PR review ✨😅